CHANGELOG.rst

```diff
@@ -2,19 +2,40 @@
 Change history
 ==============
 
-2.x.x
+2.4.0
 =====
-*TBD*
+*November, 22, 2024*
 
 **New features**
 
-* Updated OAF version to 0.9.0. This upgrade allows admin users managing their sessions through the admin.
 * [#197] Added a ``migrate_to_v2`` management command which allows users of version ``1.0.0`` to migrate to version ``2.4.0``
   More information can be found in the `documentation <https://open-klant.readthedocs.io/en/latest/installation/migration.html>`
+* [#246] Added ``is_standaard_adres`` for ``DigitaalAdres``
+* Updated OAF version to 0.9.0. This upgrade allows admin users managing their sessions through the admin.
+* [#232] Added ``/maak-klantcontact`` convenience endpoint. This allows creating
+  a ``KlantContact``, a ``Betrokkene`` and a ``OnderwerpObject`` through a
+  single API request
+* [#232] Added ``soortDigitaalAdres`` enum for ``DigitaalAdres``
 
 **Bugfixes/QoL**:
 
-* [#258] Use correct API root in redoc OAS
+* [#235] Added extra validation for phone numbers for ``DigitaalAdres.adres``
+  when ``DigitaalAdres.soortDigitaalAdres`` is ``telefoonnummer``.
+* [#243] Fix expand query parameters. Shows the ``_expand`` field in the response body
+  even though it might be empty. This behavior is applied to all available
+  ``_expand`` parameters.
+* [#258] Added correct API root paths in redoc OAS
+* [#234] Added validation for ``DigitaalAdres.adres`` whenever it's type is ``email``
+* [#227] Fixed ``partijIdentificatie`` crashing when not provided
+* [#261] Fixed ``Onderwerpobject`` inline to use ``klantcontact``
```

Comment:
```suggestion
* [#261] Fixed ``Onderwerpobject`` inline to use ``klantcontact`` instead of ``was_klantcontact``
```
Just to make it more clear. 
Might want to change inline to admin? Are these for devs or the users? 

Also I don't like the wording. It is technically a fix but I think  ``Changed`` is better.